.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -40,6 +40,14 @@ body:
     validations:
       required: true
 
+  - type: input
+    attributes:
+      label: Android device information (optional)
+      description: Please provide Android device details if the issue occurs on or is related to Android.
+      placeholder: Pixel 6 - Android 12 - Google Tensor - Mali-G78 MP20 - 8 GB RAM
```

Comment:
Also mention iOS:

```suggestion
      label: Target device information (optional)
      description: Please provide the target device details if the issue occurs on or is related to Android/iOS.
      placeholder: Pixel 6 - Android 12 - Google Tensor - Mali-G78 MP20 - 8 GB RAM, iPhone 15 Pro
```

Reply:
As someone who recently bought a Samsung device, it's not a hard rule. Some Samsung devices in my corner of EU are Snapdragons (though my personal one is Exynos)

Reply:
@syntaxerror247 I updated this comment since I first posted it, so please check it again.